### PR TITLE
use parameters so that queries are easier to read fixes #345

### LINKED ISSUE #345: apoc.es - Url encoding on query parameters should be done only on values
If I perform a rest call to ES with any query parameters (i.e. "?q=name:Neo4j&_source_include=name") all the query string is url-encoded while only "name:Neo4j" and "name" should be encoded.

### PR DESCRIPTION
**Pull request by albertodelazzari**
use parameters so that queries are easier to read fixes #345

### CODE REVIEW

File: src/test/java/apoc/es/ElasticSearchTest.java [hunk 1]
```diff
@@ -33,6 +33,16 @@
     private final static String HOST = "localhost";
 
     protected static GraphDatabaseService db;
+
+    private static Map<String, Object> defaultParams = new HashMap<>();
```
Comment (jexp) [inline src/test/java/apoc/es/ElasticSearchTest.java]: Util.map()

File: src/test/java/apoc/es/ElasticSearchTest.java [hunk 2]
```diff
@@ -53,25 +63,37 @@ public static void tearDown() {
...
     }
 
+    /**
+     * Default params (host, index, type, id) + payload
+     *
+     * @param payload
+     * @return
+     */
+    private static Map<String, Object> createDefaultProcedureParametersWithPayloadAndId(String payload, String id) {
+        Map<String, Object> params = new HashMap<>(defaultParams);
```
Comment (jexp) [inline src/test/java/apoc/es/ElasticSearchTest.java]: `Util.merge(defaultParams, Util.map("payload",payload,"id",id))`

**Comment by albertodelazzari** [comment]
Fixes done according to your feedback.
Thanks

**Comment by jexp** [comment]
Thanks a lot

**Pull request merged**